src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #757575; // $color-gray-medium
+@pagination-bg-color: #f1f1f1; // $color-gray-lightest
+
+// Sizing variables
+
+@pagination-font-size__px: @base-font-size-px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
```

Comment:
Could possibly add a code comment here that these set the button height/width, not the full pagination component or something. Or update the names to be `@pagination-btn-height__px`